numba/tests/test_np_functions.py

```diff
@@ -296,6 +299,32 @@ def check(x_types, x_values):
         x_types = [types.complex64, types.complex128]
         check(x_types, x_values)
 
+    def test_delete(self):
+
+        def arrays():
+            # array, obj, axis
+            # 
+            # 1d array, scalar, None
+            yield np.arange(10), 3, None
```

Comment:
A test is needed with an array-like type given for the first input (`arr`), e.g. `[1,2,3]`